lib/zendesk_apps_support/location.rb

```diff
@@ -3,16 +3,16 @@ module Location
     # the numbers below match the enum values on the database, do not change them!
     LOCATIONS_AVAILABLE = {
       'zendesk' => {
-        'top_bar' => 1,
-        'nav_bar' => 2,
-        'ticket_sidebar' => 3,
-        'new_ticket_sidebar' => 4,
-        'user_sidebar' => 5,
-        'organization_sidebar' => 6,
-        'background' => 7
+        'top_bar' => { 'id' => 1, 'orderable' => true },
+        'nav_bar' => { 'id' => 2, 'orderable' => true },
+        'ticket_sidebar' => { 'id' => 3, 'orderable' => true },
+        'new_ticket_sidebar' => { 'id' => 4, 'orderable' => true },
+        'user_sidebar' => { 'id' => 5, 'orderable' => true },
+        'organization_sidebar' => { 'id' => 6, 'orderable' => true },
+        'background' => { 'id' => 7, 'orderable' => false }
       },
       'zopim' => {
-        'chat_sidebar' => 8
+        'chat_sidebar' => { 'id' => 8, 'orderable' => false }
```

Comment:
the code doesn't know that. when we have multiple zopim apps we'd have to come back and update this... seems like a waste if we already have host-level filtering